test/Interop/Cxx/static/global-var-irgen.swift

```diff
@@ -0,0 +1,13 @@
+// Test that global variables are handled properly by the ClangImporter.
+
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -I %S/Inputs -emit-ir -o %t/global-var.ir -Xfrontend -enable-cxx-interop
+// RUN: %FileCheck < %t/global-var.ir %s
+
+// CHECK: @counter = external global i32, align 4
```

Comment:
Do you want to also test the code generation pattern for the read and write into the variable?